2011-11-11  Ulrich Drepper  <drepper@gmail.com>

	[BZ #13147]
	* locales/de_AT: Use de_DE for LC_NUMERIC.
	* locales/es_BO: Use es_ES for LC_MESSAGES and LC_NUMERIC.
	* locales/es_CL: Likewise.
	* locales/es_CO: Likewise.
	* locales/es_DO: Likewise.
	* locales/es_EC: Likewise.
	* locales/es_GT: Likewise.
	* locales/es_HN: Likewise.
	* locales/es_MX: Likewise.
	* locales/es_NI: Likewise.
	* locales/es_PA: Likewise.
	* locales/es_PE: Likewise.
	* locales/es_PR: Likewise.
	* locales/es_PY: Likewise.
	* locales/es_SV: Likewise.
	* locales/es_UY: Likewise.
	* locales/es_VE: Likewise.
	* locales/es_ES: Fix LC_NUMERIC.
	* locales/es_CR: Use es_ES for LC_MESSAGES.
	* locales/fr_BE: Actually use grouping in LC_NUMERIC.
	* locales/fr_CA: Use fr_FR for LC_MESSAGES and LC_NUMERIC.
	* locales/fr_CH: Use de_CH for LC_NUMERIC.
	* locales/fr_LU: Use same grouping in LC_NUMERIC as in LC_MONETARY.

2010-05-17  Andreas Schwab  <schwab@redhat.com>

	* locales/es_CR (LC_ADDRESS): Fix character names in lang_ab.

Index: glibc-2.12-2-gc4ccff1/localedata/locales/de_AT
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/de_AT
+++ glibc-2.12-2-gc4ccff1/localedata/locales/de_AT
@@ -78,41 +78,39 @@ n_sign_posn               1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point             "<U002C>"
-thousands_sep             ""
-grouping                  0;0
+copy "de_DE"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0053><U006F><U006E>";"<U004D><U006F><U006E>";/
-        "<U0044><U0069><U0065>";"<U004D><U0069><U0074>";/
-        "<U0044><U006F><U006E>";"<U0046><U0072><U0065>";/
-        "<U0053><U0061><U006D>"
+	"<U0044><U0069><U0065>";"<U004D><U0069><U0074>";/
+	"<U0044><U006F><U006E>";"<U0046><U0072><U0065>";/
+	"<U0053><U0061><U006D>"
 day     "<U0053><U006F><U006E><U006E><U0074><U0061><U0067>";/
-        "<U004D><U006F><U006E><U0074><U0061><U0067>";/
-        "<U0044><U0069><U0065><U006E><U0073><U0074><U0061><U0067>";/
-        "<U004D><U0069><U0074><U0074><U0077><U006F><U0063><U0068>";/
-        "<U0044><U006F><U006E><U006E><U0065><U0072><U0073><U0074><U0061><U0067>";/
-        "<U0046><U0072><U0065><U0069><U0074><U0061><U0067>";/
-        "<U0053><U0061><U006D><U0073><U0074><U0061><U0067>"
+	"<U004D><U006F><U006E><U0074><U0061><U0067>";/
+	"<U0044><U0069><U0065><U006E><U0073><U0074><U0061><U0067>";/
+	"<U004D><U0069><U0074><U0074><U0077><U006F><U0063><U0068>";/
+	"<U0044><U006F><U006E><U006E><U0065><U0072><U0073><U0074><U0061><U0067>";/
+	"<U0046><U0072><U0065><U0069><U0074><U0061><U0067>";/
+	"<U0053><U0061><U006D><U0073><U0074><U0061><U0067>"
 abmon   "<U004A><U00E4><U006E>";"<U0046><U0065><U0062>";/
-        "<U004D><U00E4><U0072>";"<U0041><U0070><U0072>";/
-        "<U004D><U0061><U0069>";"<U004A><U0075><U006E>";/
-        "<U004A><U0075><U006C>";"<U0041><U0075><U0067>";/
-        "<U0053><U0065><U0070>";"<U004F><U006B><U0074>";/
-        "<U004E><U006F><U0076>";"<U0044><U0065><U007A>"
+	"<U004D><U00E4><U0072>";"<U0041><U0070><U0072>";/
+	"<U004D><U0061><U0069>";"<U004A><U0075><U006E>";/
+	"<U004A><U0075><U006C>";"<U0041><U0075><U0067>";/
+	"<U0053><U0065><U0070>";"<U004F><U006B><U0074>";/
+	"<U004E><U006F><U0076>";"<U0044><U0065><U007A>"
 mon     "<U004A><U00E4><U006E><U006E><U0065><U0072>";/
-        "<U0046><U0065><U0062><U0065><U0072>";/
-        "<U004D><U00E4><U0072><U007A>";/
-        "<U0041><U0070><U0072><U0069><U006C>";/
-        "<U004D><U0061><U0069>";/
-        "<U004A><U0075><U006E><U0069>";/
-        "<U004A><U0075><U006C><U0069>";/
-        "<U0041><U0075><U0067><U0075><U0073><U0074>";/
-        "<U0053><U0065><U0070><U0074><U0065><U006D><U0062><U0065><U0072>";/
-        "<U004F><U006B><U0074><U006F><U0062><U0065><U0072>";/
-        "<U004E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
-        "<U0044><U0065><U007A><U0065><U006D><U0062><U0065><U0072>"
+	"<U0046><U0065><U0062><U0065><U0072>";/
+	"<U004D><U00E4><U0072><U007A>";/
+	"<U0041><U0070><U0072><U0069><U006C>";/
+	"<U004D><U0061><U0069>";/
+	"<U004A><U0075><U006E><U0069>";/
+	"<U004A><U0075><U006C><U0069>";/
+	"<U0041><U0075><U0067><U0075><U0073><U0074>";/
+	"<U0053><U0065><U0070><U0074><U0065><U006D><U0062><U0065><U0072>";/
+	"<U004F><U006B><U0074><U006F><U0062><U0065><U0072>";/
+	"<U004E><U006F><U0076><U0065><U006D><U0062><U0065><U0072>";/
+	"<U0044><U0065><U007A><U0065><U006D><U0062><U0065><U0072>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0059><U002D><U0025><U006D><U002D><U0025><U0064>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_BO
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_BO
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_BO
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_CL
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_CL
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_CL
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_CO
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_CO
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_CO
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_CR
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_CR
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_CR
@@ -47,8 +47,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,34 +78,34 @@ END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0059>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_DO
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_DO
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_DO
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_EC
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_EC
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_EC
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_ES
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_ES
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_ES
@@ -85,40 +85,40 @@ END LC_MONETARY
 
 LC_NUMERIC
 decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+thousands_sep        "<U002E>"
+grouping             3;3
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_GT
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_GT
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_GT
@@ -55,8 +55,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -78,9 +77,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_HN
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_HN
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_HN
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_MX
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_MX
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_MX
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_NI
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_NI
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_NI
@@ -47,8 +47,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -70,9 +69,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_PA
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_PA
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_PA
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_PE
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_PE
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_PE
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_PR
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_PR
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_PR
@@ -47,8 +47,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -70,41 +69,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_PY
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_PY
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_PY
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_SV
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_SV
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_SV
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,9 +78,7 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002E>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_UY
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_UY
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_UY
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/es_VE
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/es_VE
+++ glibc-2.12-2-gc4ccff1/localedata/locales/es_VE
@@ -56,8 +56,7 @@ copy "es_ES"
 END LC_CTYPE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U0073><U0053><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "es_ES"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -79,41 +78,39 @@ n_sign_posn          1
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point        "<U002C>"
-thousands_sep        ""
-grouping             0;0
+copy "es_ES"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U006F><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
-        "<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
-        "<U0073><U00E1><U0062>"
+	"<U006D><U0061><U0072>";"<U006D><U0069><U00E9>";/
+	"<U006A><U0075><U0065>";"<U0076><U0069><U0065>";/
+	"<U0073><U00E1><U0062>"
 day     "<U0064><U006F><U006D><U0069><U006E><U0067><U006F>";/
-        "<U006C><U0075><U006E><U0065><U0073>";/
-        "<U006D><U0061><U0072><U0074><U0065><U0073>";/
-        "<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
-        "<U006A><U0075><U0065><U0076><U0065><U0073>";/
-        "<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
-        "<U0073><U00E1><U0062><U0061><U0064><U006F>"
+	"<U006C><U0075><U006E><U0065><U0073>";/
+	"<U006D><U0061><U0072><U0074><U0065><U0073>";/
+	"<U006D><U0069><U00E9><U0072><U0063><U006F><U006C><U0065><U0073>";/
+	"<U006A><U0075><U0065><U0076><U0065><U0073>";/
+	"<U0076><U0069><U0065><U0072><U006E><U0065><U0073>";/
+	"<U0073><U00E1><U0062><U0061><U0064><U006F>"
 abmon   "<U0065><U006E><U0065>";"<U0066><U0065><U0062>";/
-        "<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
-        "<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0062><U0072>";/
+	"<U006D><U0061><U0079>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U006C>";"<U0061><U0067><U006F>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U0069><U0063>"
 mon     "<U0065><U006E><U0065><U0072><U006F>";/
-        "<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
-        "<U006D><U0061><U0072><U007A><U006F>";/
-        "<U0061><U0062><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0079><U006F>";/
-        "<U006A><U0075><U006E><U0069><U006F>";/
-        "<U006A><U0075><U006C><U0069><U006F>";/
-        "<U0061><U0067><U006F><U0073><U0074><U006F>";/
-        "<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U0065><U0062><U0072><U0065><U0072><U006F>";/
+	"<U006D><U0061><U0072><U007A><U006F>";/
+	"<U0061><U0062><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0079><U006F>";/
+	"<U006A><U0075><U006E><U0069><U006F>";/
+	"<U006A><U0075><U006C><U0069><U006F>";/
+	"<U0061><U0067><U006F><U0073><U0074><U006F>";/
+	"<U0073><U0065><U0070><U0074><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U0075><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0069><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U0069><U0063><U0069><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002F><U0025><U006D><U002F><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/fr_BE
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/fr_BE
+++ glibc-2.12-2-gc4ccff1/localedata/locales/fr_BE
@@ -79,7 +79,7 @@ END LC_MONETARY
 LC_NUMERIC
 decimal_point             "<U002C>"
 thousands_sep             "<U002E>"
-grouping                  0;0
+grouping                  3;3
 END LC_NUMERIC
 
 LC_TIME
Index: glibc-2.12-2-gc4ccff1/localedata/locales/fr_CA
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/fr_CA
+++ glibc-2.12-2-gc4ccff1/localedata/locales/fr_CA
@@ -55,8 +55,7 @@ copy "en_CA"
 END LC_COLLATE
 
 LC_MESSAGES
-yesexpr "<U005E><U005B><U006F><U004F><U0079><U0059><U005D><U002E><U002A>"
-noexpr  "<U005E><U005B><U006E><U004E><U005D><U002E><U002A>"
+copy "fr_FR"
 END LC_MESSAGES
 
 LC_MONETARY
@@ -78,41 +77,39 @@ n_sign_posn               0
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point             "<U002C>"
-thousands_sep             "<U0020>"
-grouping                  0;0
+copy "fr_FR"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U0069><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
-        "<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
-        "<U0073><U0061><U006D>"
+	"<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
+	"<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
+	"<U0073><U0061><U006D>"
 day     "<U0064><U0069><U006D><U0061><U006E><U0063><U0068><U0065>";/
-        "<U006C><U0075><U006E><U0064><U0069>";/
-        "<U006D><U0061><U0072><U0064><U0069>";/
-        "<U006D><U0065><U0072><U0063><U0072><U0065><U0064><U0069>";/
-        "<U006A><U0065><U0075><U0064><U0069>";/
-        "<U0076><U0065><U006E><U0064><U0072><U0065><U0064><U0069>";/
-        "<U0073><U0061><U006D><U0065><U0064><U0069>"
+	"<U006C><U0075><U006E><U0064><U0069>";/
+	"<U006D><U0061><U0072><U0064><U0069>";/
+	"<U006D><U0065><U0072><U0063><U0072><U0065><U0064><U0069>";/
+	"<U006A><U0065><U0075><U0064><U0069>";/
+	"<U0076><U0065><U006E><U0064><U0072><U0065><U0064><U0069>";/
+	"<U0073><U0061><U006D><U0065><U0064><U0069>"
 abmon   "<U006A><U0061><U006E>";"<U0066><U00E9><U0076>";/
-        "<U006D><U0061><U0072>";"<U0061><U0076><U0072>";/
-        "<U006D><U0061><U0069>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U0069>";"<U0061><U006F><U00FB>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U00E9><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0076><U0072>";/
+	"<U006D><U0061><U0069>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U0069>";"<U0061><U006F><U00FB>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U00E9><U0063>"
 mon     "<U006A><U0061><U006E><U0076><U0069><U0065><U0072>";/
-        "<U0066><U00E9><U0076><U0072><U0069><U0065><U0072>";/
-        "<U006D><U0061><U0072><U0073>";/
-        "<U0061><U0076><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0069>";/
-        "<U006A><U0075><U0069><U006E>";/
-        "<U006A><U0075><U0069><U006C><U006C><U0065><U0074>";/
-        "<U0061><U006F><U00FB><U0074>";/
-        "<U0073><U0065><U0070><U0074><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U006F><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U00E9><U0063><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U00E9><U0076><U0072><U0069><U0065><U0072>";/
+	"<U006D><U0061><U0072><U0073>";/
+	"<U0061><U0076><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0069>";/
+	"<U006A><U0075><U0069><U006E>";/
+	"<U006A><U0075><U0069><U006C><U006C><U0065><U0074>";/
+	"<U0061><U006F><U00FB><U0074>";/
+	"<U0073><U0065><U0070><U0074><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U006F><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U00E9><U0063><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0059><U002D><U0025><U006D><U002D><U0025><U0064>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/fr_CH
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/fr_CH
+++ glibc-2.12-2-gc4ccff1/localedata/locales/fr_CH
@@ -64,41 +64,39 @@ copy  "de_CH"
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point             "<U002C>"
-thousands_sep             ""
-grouping                  0;0
+copy "de_CH"
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U0069><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
-        "<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
-        "<U0073><U0061><U006D>"
+	"<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
+	"<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
+	"<U0073><U0061><U006D>"
 day     "<U0064><U0069><U006D><U0061><U006E><U0063><U0068><U0065>";/
-        "<U006C><U0075><U006E><U0064><U0069>";/
-        "<U006D><U0061><U0072><U0064><U0069>";/
-        "<U006D><U0065><U0072><U0063><U0072><U0065><U0064><U0069>";/
-        "<U006A><U0065><U0075><U0064><U0069>";/
-        "<U0076><U0065><U006E><U0064><U0072><U0065><U0064><U0069>";/
-        "<U0073><U0061><U006D><U0065><U0064><U0069>"
+	"<U006C><U0075><U006E><U0064><U0069>";/
+	"<U006D><U0061><U0072><U0064><U0069>";/
+	"<U006D><U0065><U0072><U0063><U0072><U0065><U0064><U0069>";/
+	"<U006A><U0065><U0075><U0064><U0069>";/
+	"<U0076><U0065><U006E><U0064><U0072><U0065><U0064><U0069>";/
+	"<U0073><U0061><U006D><U0065><U0064><U0069>"
 abmon   "<U006A><U0061><U006E>";"<U0066><U00E9><U0076>";/
-        "<U006D><U0061><U0072>";"<U0061><U0076><U0072>";/
-        "<U006D><U0061><U0069>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U0069>";"<U0061><U006F><U00FB>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U00E9><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0076><U0072>";/
+	"<U006D><U0061><U0069>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U0069>";"<U0061><U006F><U00FB>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U00E9><U0063>"
 mon     "<U006A><U0061><U006E><U0076><U0069><U0065><U0072>";/
-        "<U0066><U00E9><U0076><U0072><U0069><U0065><U0072>";/
-        "<U006D><U0061><U0072><U0073>";/
-        "<U0061><U0076><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0069>";/
-        "<U006A><U0075><U0069><U006E>";/
-        "<U006A><U0075><U0069><U006C><U006C><U0065><U0074>";/
-        "<U0061><U006F><U00FB><U0074>";/
-        "<U0073><U0065><U0070><U0074><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U006F><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U00E9><U0063><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U00E9><U0076><U0072><U0069><U0065><U0072>";/
+	"<U006D><U0061><U0072><U0073>";/
+	"<U0061><U0076><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0069>";/
+	"<U006A><U0075><U0069><U006E>";/
+	"<U006A><U0075><U0069><U006C><U006C><U0065><U0074>";/
+	"<U0061><U006F><U00FB><U0074>";/
+	"<U0073><U0065><U0070><U0074><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U006F><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U00E9><U0063><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002E><U0020><U0025><U006D><U002E><U0020><U0025><U0079>"
 t_fmt   "<U0025><U0054>"
Index: glibc-2.12-2-gc4ccff1/localedata/locales/fr_LU
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/localedata/locales/fr_LU
+++ glibc-2.12-2-gc4ccff1/localedata/locales/fr_LU
@@ -78,40 +78,40 @@ END LC_MONETARY
 
 LC_NUMERIC
 decimal_point             "<U002C>"
-thousands_sep             ""
-grouping                  0;0
+thousands_sep             "<U0020>"
+grouping                  3;3
 END LC_NUMERIC
 
 LC_TIME
 abday   "<U0064><U0069><U006D>";"<U006C><U0075><U006E>";/
-        "<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
-        "<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
-        "<U0073><U0061><U006D>"
+	"<U006D><U0061><U0072>";"<U006D><U0065><U0072>";/
+	"<U006A><U0065><U0075>";"<U0076><U0065><U006E>";/
+	"<U0073><U0061><U006D>"
 day     "<U0064><U0069><U006D><U0061><U006E><U0063><U0068><U0065>";/
-        "<U006C><U0075><U006E><U0064><U0069>";/
-        "<U006D><U0061><U0072><U0064><U0069>";/
-        "<U006D><U0065><U0072><U0063><U0072><U0065><U0064><U0069>";/
-        "<U006A><U0065><U0075><U0064><U0069>";/
-        "<U0076><U0065><U006E><U0064><U0072><U0065><U0064><U0069>";/
-        "<U0073><U0061><U006D><U0065><U0064><U0069>"
+	"<U006C><U0075><U006E><U0064><U0069>";/
+	"<U006D><U0061><U0072><U0064><U0069>";/
+	"<U006D><U0065><U0072><U0063><U0072><U0065><U0064><U0069>";/
+	"<U006A><U0065><U0075><U0064><U0069>";/
+	"<U0076><U0065><U006E><U0064><U0072><U0065><U0064><U0069>";/
+	"<U0073><U0061><U006D><U0065><U0064><U0069>"
 abmon   "<U006A><U0061><U006E>";"<U0066><U00E9><U0076>";/
-        "<U006D><U0061><U0072>";"<U0061><U0076><U0072>";/
-        "<U006D><U0061><U0069>";"<U006A><U0075><U006E>";/
-        "<U006A><U0075><U0069>";"<U0061><U006F><U00FB>";/
-        "<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
-        "<U006E><U006F><U0076>";"<U0064><U00E9><U0063>"
+	"<U006D><U0061><U0072>";"<U0061><U0076><U0072>";/
+	"<U006D><U0061><U0069>";"<U006A><U0075><U006E>";/
+	"<U006A><U0075><U0069>";"<U0061><U006F><U00FB>";/
+	"<U0073><U0065><U0070>";"<U006F><U0063><U0074>";/
+	"<U006E><U006F><U0076>";"<U0064><U00E9><U0063>"
 mon     "<U006A><U0061><U006E><U0076><U0069><U0065><U0072>";/
-        "<U0066><U00E9><U0076><U0072><U0069><U0065><U0072>";/
-        "<U006D><U0061><U0072><U0073>";/
-        "<U0061><U0076><U0072><U0069><U006C>";/
-        "<U006D><U0061><U0069>";/
-        "<U006A><U0075><U0069><U006E>";/
-        "<U006A><U0075><U0069><U006C><U006C><U0065><U0074>";/
-        "<U0061><U006F><U00FB><U0074>";/
-        "<U0073><U0065><U0070><U0074><U0065><U006D><U0062><U0072><U0065>";/
-        "<U006F><U0063><U0074><U006F><U0062><U0072><U0065>";/
-        "<U006E><U006F><U0076><U0065><U006D><U0062><U0072><U0065>";/
-        "<U0064><U00E9><U0063><U0065><U006D><U0062><U0072><U0065>"
+	"<U0066><U00E9><U0076><U0072><U0069><U0065><U0072>";/
+	"<U006D><U0061><U0072><U0073>";/
+	"<U0061><U0076><U0072><U0069><U006C>";/
+	"<U006D><U0061><U0069>";/
+	"<U006A><U0075><U0069><U006E>";/
+	"<U006A><U0075><U0069><U006C><U006C><U0065><U0074>";/
+	"<U0061><U006F><U00FB><U0074>";/
+	"<U0073><U0065><U0070><U0074><U0065><U006D><U0062><U0072><U0065>";/
+	"<U006F><U0063><U0074><U006F><U0062><U0072><U0065>";/
+	"<U006E><U006F><U0076><U0065><U006D><U0062><U0072><U0065>";/
+	"<U0064><U00E9><U0063><U0065><U006D><U0062><U0072><U0065>"
 d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
 d_fmt   "<U0025><U0064><U002E><U0025><U006D><U002E><U0025><U0059>"
 t_fmt   "<U0025><U0054>"
